README.md

```diff
@@ -1,94 +1,79 @@
-This was the result of a discovery sprint, I am in the process of cleaning it up for easier local install.
-
 ## Local set up
 
-This is using Pipenv and Postgres
-
-### Pipenv info
-
- https://github.com/pypa/pipenv
-
-### Postgres
-
- https://www.postgresql.org/download/
-
-I recommend [pgcli](https://www.pgcli.com/) for interfacing with postgres
+Install Docker
 
+    https://www.docker.com/get-started
 
-In pgcli or your sql shell command to create datebase with:
 
-    create database crt_portal;
+Use a long, random string for your secret key and save it in your local environment.
 
+    export SECRET_KEY=''
 
-commands to create local db user for app, replace `username` and `userpass` with your choices:
+To build the project
 
-    CREATE USER username WITH PASSWORD 'userpass' CREATEDB;
-    GRANT permissions ON DATABASE crt_portal TO username;
+    docker-compose up -d --build
 
+To run the project
 
-(I would like the local set up to be in docker next. so we don't need to to the previous steps)
+    docker-compose u
```

Comment:
@LindsayYoung Should this be `docker-compose up`?

Reply:
yep!